modalMain.js

```diff
@@ -0,0 +1,11 @@
+// let btnopen=document.querySelectorAll('button');
+// let myModal=document.getElementById('modal-container');
+// let btnclose=document.getElementsByClassName('fa-times');
+
+// Array.prototype.map.call(btnopen, (b) =>{
+//   b.addEventListener('click', function(){
+//     // myModal.classList.add('show');
+//     myModal.classList.toggle('show');
+//     document.body.style.overflowY = 'hidden';
+//   });
```

Comment:
Please remove all commented code from your codebase as requested by the previous code reviewer to make your codebase clean and easy to maintain.